yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -835,13 +890,18 @@ def _ffmpeg_args_for_chapter(self, number, chapter, info):
     def run(self, info):
         chapters = info.get('chapters') or []
         if not chapters:
-            self.report_warning('Chapter information is unavailable')
+            self.to_screen('Chapter information is unavailable')
             return [], info
 
+        in_file = info['filepath']
+        if self._force_keyframes:
+            in_file = self.reencode_with_keyframes_at(in_file, *(chapter['start_time'] for chapter in chapters))
```

Comment:
Might be worth skipping the start of very first chapter (00:00 most likely already has keyframes) and protecting against single entry `chapters`.